cli/rollup.config.js

```diff
@@ -0,0 +1,45 @@
+import resolve from '@rollup/plugin-node-resolve';
+import cjs from '@rollup/plugin-commonjs';
+import asset from './lib/asset-plugin.js';
+import json from './lib/json-plugin.js';
+import autojson from './lib/autojson-plugin.js';
+import { getBabelOutputPlugin } from '@rollup/plugin-babel';
+import { builtinModules } from 'module';
+
+/** @type {import('rollup').RollupOptions} */
+export default {
+  input: 'src/index.js',
+  output: {
+    dir: 'build',
+    format: 'cjs',
+    assetFileNames: '[name]-[hash][extname]',
+    // This is needed so the resulting `index.js` can be
+    // executed by `npx`.
+    banner: '#!/usr/bin/env node',
```

Comment:
Hmm, I thought `bin` field in `package.json` is enough? Otherwise, how would `npx` execute it on Windows where shebang is ignored?

Reply:
I didn’t work without this. Maybe you can give `npx @squoosh/cli` a quick test on your Windows machine?

Reply:
Seems to work.